iphone/Maps/Classes/Widgets/MWMMapDownloadDialog.mm

```diff
@@ -85,6 +86,7 @@ - (void)configDialog {
     self.nodeSize.hidden = NO;
     self.nodeSize.textColor = [UIColor blackSecondaryText];
     self.nodeSize.text = formattedSize(nodeAttrs.m_mwmSize);
+    self.nodeSize.font = [self monospacedFontWithSize:14.0];
```

Comment:
Should the size be inherited/derived from the existing/default font?

Reply:
In this case, it's important to remove the runtime KVC attribute from the storyboard.
The setting will be set programmatically (one source of truth) and the textColor / font won't be duplicated in two places.

<img width="400" alt="image" src="https://github.com/user-attachments/assets/7dc1e6b8-3cfa-4cd9-b6ce-e5cdcda20328">

Also don't forget to use the medium font.

Reply:
@kirylkaveryn thanks. Its a valuable point. Removed this property.